ClickHaskell-tables/lib/ClickHaskell/Tables.hs

```diff
@@ -161,132 +164,34 @@ renderTableParameters :: [Builder] -> Builder
 renderTableParameters (parameter:ps) = "(" <> foldr (\p1 p2 -> p1 <> ", " <> p2) parameter ps <> ")"
 renderTableParameters []             = ""
 
+data PList (ps :: [Type]) where
+  PNil :: PList '[]
+  (:#) :: (p ~ Parameter chName chType, ToParameterList ps) => p -> PList ps -> PList (p ': ps)
```

Comment:
### About `PNil` existence
From View description we could know about cases when there is no parameters. Is there are any cases when we need to have `PNil`?

### About `:#` name choice
In my opinion operators such as `:#` is hard to remember and understand. I don't see any problems to use some function with name like `parameter`/`pass`/`passParameter` for such important part of logic as arguments passing, During refactoring it's nice to understand properly what part of code means when you are not familiar with ClickHaskell API